bindings/nodejs/README.md

```diff
@@ -64,9 +127,23 @@ async function run() {
 run().then(() => process.exit());
 ```
 
+#### Examples
+
+You can use the provided code [examples](examples) to acquainted with the IOTA SDK. You can use the following command to run any example:
+
+```bash
+node example/[example file]
```

Comment:
lets switch this to yarn, as cloning from source requires that (We use `link: ../`, which isnt supported in npm)

```suggestion
cd examples
yarn run-example ./[example folder]/[example file]
```

andf or example:
`yarn run-example ./client/00_get_info.ts`